### PR TITLE
Add region page styling

### PR DESCRIPTION
**Pull request by noelcastillon1**
This addresses #351 <3

### CODE REVIEW

**Comment by ghost** [comment]
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=704&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=704&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org)
</details>

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=704&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org)

#### Legend
CodeSee Map Legend

File: src/components/list/UpdateList.tsx [hunk 1]
```diff
@@ -20,9 +20,9 @@ const LinkList: FC<Props> = ({ list: { title, visibleCount, updates } }) => {
         {updates.map(({ title, date, content }) => (
           <li className="p-7">
             <h3 className="text-center text-2xl">{title}</h3>
-            <p className="py-4 text-center">{date}</p>
+            <p className="py-4 text-center">{new Date(date).toUTCString()}</p>
             <div className="w-full">
-              <article className="prose mx-auto mb-4 max-w-xl py-2 text-center  text-gray-800 sm:text-lg">
+              <article className="prose mx-auto mb-4 max-w-xl py-2 text-center text-gray-800 sm:text-lg">
```
Comment (hola-soy-milk) [inline src/components/list/UpdateList.tsx:25]: Thank you for removing this, it was bugging me!

File: src/components/list/UpdateList.tsx [hunk 2]
```diff
@@ -20,9 +20,9 @@ const LinkList: FC<Props> = ({ list: { title, visibleCount, updates } }) => {
         {updates.map(({ title, date, content }) => (
           <li className="p-7">
             <h3 className="text-center text-2xl">{title}</h3>
-            <p className="py-4 text-center">{date}</p>
+            <p className="py-4 text-center">{new Date(date).toUTCString()}</p>
```
Comment (hola-soy-milk) [inline src/components/list/UpdateList.tsx:23]: Yay, international!

**Review by hola-soy-milk — APPROVED** [review]
Thank you so much, @noelcastillon1! This looks loads better and consistent with the rest of the page! 🎉

**Pull request merged**